src/modules/launcher/Wox.Core/Resource/FontHelper.cs

```diff
@@ -24,7 +25,9 @@ public static FontWeight GetFontWeightFromInvariantStringOrNormal(string value)
             {
                 return (FontWeight)_fontWeightConverter.ConvertFromInvariantString(value);
             }
+#pragma warning disable CA1031 // Do not catch general exception types
```

Comment:
We should try to understand what exception we are catching, as this could silently fail due to programmer error.  At the very least we should be logging the exception.